ProfileManager.js

```diff
@@ -84,6 +92,118 @@ export default class ProfileManager {
     await this._sessionChanged({newData: session.data});
   }
 
+  // TODO: add docs
+  // FIXME: rename?
+  // create a capabilitySet EDV and create a document in that EDV and create
+  // delegations related to that document
+  async createCapabilitySetEdv({profileId, referenceId, zcaps}) {
+    const {profileAgent} = await this._profileService.getAgentByProfile({
+      account: this.accountId,
+      profile: profileId
+    });
+    const {invocationSigner, kmsClient} = await this.getProfileSigner({
+      profileId,
+      profileAgentId: profileAgent.id
+    });
+    const edv = await edvs.create({
+      invocationSigner,
+      kmsClient,
+      kmsModule: this.kmsModule,
+      profileId,
+      referenceId,
+      edvBaseUrl: this.edvBaseUrl,
+    });
+
+    // create the capabilitySet document for the profile agent
+    const capabilitySetDocument = await edv.insert({
+      doc: {
+        content: {
+          // FIXME:
+          // WHAT OTHER PROPERTIES ARE NEEDED
+          // WE MAY NEED DETAILS THAT ASSIST IN IDENTIFYING CAPABILITYSET DOCS
+          // FOR THE PURPOSE OF DELETING THEM FOR INSTANCE.
```

Comment:
The profile agent ID could go in here. The profile ID could as well if we feel that's necessary, but the doc and EDV belong to the profile.